modules/programs/starship.nix

```diff
@@ -89,6 +89,14 @@ in {
         Whether to enable Ion integration.
       '';
     };
+
+    enableNushellIntegration = mkOption {
+      default = true;
+      type = types.bool;
+      description = ''
+        Whether to enable Nushell integration.
+      '';
+    };
```

Comment:
```suggestion

    enableNushellIntegration = mkEnableOption "Nushell integration" // {
      default = true;
    };
```

Reply:
Will need to be fixed in the master branch first :-)

Reply:
Resolved by https://github.com/nix-community/home-manager/commit/0b0986f0c054b5e1358ab8ce2825b00897eb7da8.